Update preflight-checks.mdx (#1606)
seqeralabs/docs: platform-cloud/docs/compute-envs/preflight-checks.mdx

## What is checked and when

Seqera Platform carries out  three different tiers of validation:

### 1. Credential validation

Runs on a recurring schedule. For each cloud credential (AWS, GCP, Azure) in scope, Platform calls the provider API to verify the credential is still accepted and can authenticate. The credential sweep makes a live API call to verify the credential is still accepted by the cloud provider. For AWS role-based credentials and GCP Workload Identity Federation, this check confirms the credential is well-formed but cannot fully verify the underlying role or identity provider trust configuration.

When a credential fails this check, it is marked **INVALID** with the provider error recorded on the credential record. This error appears in the launch-time error when a pipeline is blocked, but not in the CE banner — to see the specific provider error, check the credential record directly.

### 2. Compute environment validation

Runs approximately every hour across all compute environments where the status is **AVAILABLE**. The checks validate:

> 1. **Credential status** — If the credential is **INVALID**, the CE is immediately marked **INVALID**. 

Reviewer: ```suggestion
1. **Credential status**. If the credential is **INVALID**, the compute environment is immediately marked **INVALID**. 
```

> 2. **Work directory** — Validates whether the Compute environment's configured work directory is accessible. If this fails, the CE is marked **INVALID** with the provider error appended in the card view and at the top of the form page. 

Reviewer: ```suggestion
2. **Work directory**. Validates whether the compute environment's configured work directory is accessible. If this fails, the compute environment is marked **INVALID** with the provider error appended in the card view and at the top of the form page. 
```

> A CE marked **INVALID** displays a banner with the corresponding error message. A Compute environment that is marked as **AVAILABLE** has its `lastValidated` timestamp refreshed.

Reviewer: ```suggestion
A compute environment marked **INVALID** displays a banner with the corresponding error message. A compute environment that is marked as **AVAILABLE** has its `lastValidated` timestamp refreshed.
```

> | CE status | Blocks launch if the CE is marked **INVALID** |

Reviewer: ```suggestion
| Compute environment status | Blocks launch if the CE is marked **INVALID** |
```

> | Credential status | Blocks launch if the credential associated with the CE is marked **INVALID** |

Reviewer: ```suggestion
| Credential status | Blocks launch if the credential associated with the compute environment is marked **INVALID** |
```

| Work directory override | If the user provided a different work directory at launch, validates that path with the cloud provider |

> | Wave connectivity | For Compute Environments with Wave enabled, verifies the Wave service connection is active |

Reviewer: ```suggestion
| Wave connectivity | For compute environment with Wave enabled, verifies the Wave service connection is active |
```

> | Tower Agent | For HPC Compute enrionemnts, verifies a Tower Agent is online for the environment |

Reviewer: ```suggestion
| Tower Agent | For HPC compute environments, verifies a Tower Agent is online for the environment |
```

> 2. Find the CE and open its **⋮** (three-dot) dropdown menu.

Reviewer: ```suggestion
2. Find the compute environment and open its **⋮** (three-dot) dropdown menu.
```

3. Select **Validate**.

> Seqera Platform will carry out pre-flight checks and update the CE status immediately. If all checks pass, the CE returns to **AVAILABLE**.

Reviewer: ```suggestion
Seqera Platform will carry out pre-flight checks and update the compute environment status immediately. If all checks pass, the compute environment returns to **AVAILABLE**.
```
